src/fairseq2/models/conformer/convolution.py

```diff
@@ -42,6 +87,8 @@ def __init__(
             The dimensionality of the model.
         :param depthwise_kernel_size:
             The kernel size of the depthwise convolution.
+        :param depthwise_conv_is_causal:
```

Comment:
Nit: I would prefer this to be named `causal_depthwise_conv` which is a bit more aligned with the naming convention we use in other places in the code base.